appformer-kogito-bridge/src/main/java/org/appformer/kogito/bridge/client/ls/dmn/DMNLanguageServiceApiWrapper.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021. Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.appformer.kogito.bridge.client.ls.dmn;
+
+import java.util.List;
+
+import jsinterop.annotations.JsProperty;
+import jsinterop.annotations.JsType;
+import org.uberfire.workbench.model.bridge.LanguageServiceResult;
+
+@JsType(isNative = true, namespace = "window", name = "envelope")
+public class DMNLanguageServiceApiWrapper implements DMNLanguageServiceApi {
+
+    @Override
+    public native List<LanguageServiceResult> getClasses(String completion);
+
+    @Override
+    public native List<LanguageServiceResult> getMethods(String fqdn, String completion);
+
+    @Override
+    public native List<LanguageServiceResult> getAttributes(String fqdn, String completion);
+
+    @JsProperty(name = "dmnLanguageServiceApi")
```

Comment:
Hi Adriel! NIce PR! Maybe this should be called `dmnLanguageServerApi` instead? Asking because from the PR title, the name of this new API is going to be `Language Server API`, right?

Reply:
Also, I know this will only be used by DMN for now, but maybe we don't need to have the `dmn` prefix there?

Reply:
+1, considering the idea is to have a Language Server available for all the editors, I agree with @tiagobento to remove all `dmn` prefixes